pkg/external_repos/external_repos.json

```diff
@@ -1,4 +1,52 @@
 [
+    {
+        "base_url": ""
+    },
+    {
+        "base_url": ""
+    },
```

Comment:
There's a bunch of new blank base_url's here, likely by mistake

Reply:
something is up with the 'make repos-download' command.  I'd just leave this out for now and i'll take a look

Reply:
Thanks mate! I didn't realize about this! I have removed it.